autoload/health/nvim_treesitter.vim

```diff
@@ -0,0 +1,4 @@
+" Last Change: 2020 avril 21
+function! health#nvim_treesitter#check()
+  lua require 'nvim-treesitter.health'.checkhealth()
```

Comment:
in `nvim-lsp` they also have this line before:
```vim
call health#report_start('Checking language server protocol configuration')
```

Reply:
Yep, but the way they only do one section, while we will do multiple sections (one for each installed parser)